### ISSUE TITLE
Update Static page

### ISSUE THREAD

**Issue by 01101000011011110110010100001010** [Harems-io/Front-end#3]
We need the layouts updated. I will provide images and dropbox of what's needed.

**Comment by 01101000011011110110010100001010** [comment]
https://www.dropbox.com/sh/vhr05bsm2x11j8v/AACWcbjHVUmH9FaC7F3kQEPpa?dl=0&preview=Harems.pptx

https://www.dropbox.com/sh/vhr05bsm2x11j8v/AADRchTo-l6pZmtL5M6upzdda/Japan?dl=0&subfolder_nav_tracking=1

**Comment by 01101000011011110110010100001010** [comment]
@nresh  I just gave you the files let me know if you need anything else.

**Comment by nresh** [comment]
@01101000011011110110010100001010 right on, i just applied on gitcoin.

i'll try to have this complete over the weekend.

**Comment by 01101000011011110110010100001010** [comment]
@nresh  appreciate it here goes the logo also. https://www.dropbox.com/sh/vhr05bsm2x11j8v/AACWcbjHVUmH9FaC7F3kQEPpa?dl=0&preview=Harems+Logo.zip

**Comment by nresh** [comment]
@01101000011011110110010100001010 I just looked at the specs a bit and first off have to say you guys have the most interesting NFT concept I've seen.  honestly I haven't understood the appeal of NFTs thus far, but after reading this, I think this may have some appeal haha:
> Every country will have a unique Harem Girl designed by the community of OG Gentleman’s Club members.  Each country’s Harem Girl will have a total of 41 digital collectibles – 40 will be in one pose.  Of those 40 - only 4 will be topless and only 1 will be completely nude.  The one remaining NFT will be of the RARE HAREM (read more below).

but anyway, I also have a few questions:
1. which of the logo images should be used for the header and footer logos?
2. what should happen when someone submits their email? currently that email form does not do anything.  I can setup an AWS lambda function that will do something with the form submissions, but I'm not sure what to do.  the easiest thing to implement is to just send all the submitted emails to a particular email address (which I would need from you guys)
3. curious: why TImes New Roman?  I'm by no means a designer, but that's as basic as it gets haha

**Comment by nresh** [comment]
one other note: depending on the kind of stuff you want the site to do, it may be worthwhile to convert it to a NextJS app (https://nextjs.org/) - the main advantage is that you can conditionally render static, as well as dynamic server-side code, and you can actually write backend code for stuff like emailing and api routes.

migrating from Gatsby to NextJS is relatively easy, but the bigger the site gets, the harder it will be.  so it's good to consider now, before we've done much with the site.

do you have any idea of any future additional functionality that may be on the site (outside of submission forms)?

**Comment by 01101000011011110110010100001010** [comment]
1. Use the pink logo, please. 
2. Set it up so we capture it, please.  We want to set up email communication, but it just needs to be in a place to mail folks later.  
3. Just because we wanted consistency.  It isn’t sexy, but it’s legible and a known quantity. 

After the drop, we want to build a Gentle Men's club area and OG Gentle Men's Club area with special benefits. I can't wait to see what this community turns into. 


also would it be possible for the emails to get stored in a database if not we can have them sent to a email.

**Comment by nresh** [comment]
@01101000011011110110010100001010 there is a PR with the asset/styling changes: https://github.com/Harems-io/Front-end/pull/4 

you can see the new version of the site at this temporary branch url: https://3-styling-and-asset-updates-to-landing-page.dqlawzyoy6p51.amplifyapp.com/

I've tried to make it as close as possible to the powerpoint, but lmk if you'd like any tweaks.  also, take a look both on desktop and mobile.  since I only have desktop designs to go off of, the mobile version is just a standard 

notes:
1. the image for "The Rare Harem" section is blurry since I didn't have a high res version of that one - I just took that one directly from the powerpoint, and it is low quality - send me a high quality of that one when you get the chance
2. in general, for simple logos like the ones you sent, esp those made of just text, it's preferable to use SVG files rather than pngs because svgs of logos generally have perfect resolution at a fraction of the file size of pngs.
3. lmk what you guys wanna do for the favicon - I just cropped out "HA" out of the logo and created a favicon out of that, it looks alright
4. I reduced the size of the images to improve loading, but I may reduce them even more if site speed becomes an issue.  atm the images are lazy-loaded and so the images load one at a time.  they should load fine unless you have a pretty slow phone connection
5. the form still doesn't do anything yet cause I wanted some clarifications, and also I think it should be part of a different PR. specifically, do you guys have a database where you'd want me to send it?  I can create one myself for now and just use that, and later we can migrate.  is there a preference for mysql or postgres or mongo?

also, btw, if you guys need backend help, feel free to pass stuff by me.

**Comment by 01101000011011110110010100001010** [comment]
INTRODUCTION TO HAREMS
Harems is a collection of 10,000 Non-Fungible Tokens (NFTs); each NFT is a unique digital collectible of Harem Girls covering 235 countries around the world that exist on the Ethereum blockchain.
Within those 10,000 unique NFTs – there is one unique Harem Girl per country in one of 10 different positions.  

Each Harem Girl will have traits from various standard categories including country, position, type of girl, hair color, hair style, hair accessories, background color, logo color, top clothes, bottom clothes, clothes color, shoe color and more.  

In addition to standard traits - each girl type i.e. sugar baby etc will have unique traits that can be unlocked.  An example would be only escorts can be found dead and only a trophy wife gets diamonds.  Countries who have large numbers of Muslims will have the ability to get some women with face coverings whereas countries with large numbers of Christians unlock the ability to have nuns.  Each country is able to have a maximum of one super rare trait; for example - one Saudi girl will have a picture of Osama Bin Laden in her room whereas one Japanese girl will have a picture of Pat Morina.  All of these traits are randomly selected using random equations.  

All Harem girls will be distributed on IPFS to ensure their life cycle. Ownership of any Harem girl NFT will grant access to the Gentleman’s Club; future areas and perks can be unlocked by the community with roadmap activation.

235 COUNTRIES REPRESENTED
Every country will have a unique Harem Girl designed by the community of OG Gentleman’s Club members. The amount of pictures represented by each country are determined by random algorithms.  Each country’s Harem Girl will be represented based on geographic standards; Brazilian girls are going to look Brazilian and Kenyan girls will look Kenyan.  Depending on the country - there could be unique characteristics in a picture that relate to a country’s history or background.  Gentleman love knowledge as well as beautiful women.  

10 DIFFERENT TYPES
Each Harem Girl will have unique NFTs shown for each of the following types of girl: 

Make sure the categories show properly (on text two word types are shown on two lines each)

Delete this:

And if you’re doing the math (and I know you are because gentlemen know their arithmetic) – that means there's one remaining NFT for each Harem Girl.

Delete: The Rare Harem section is gone

OG Gentleman’s Club: make it 235 countries


Also, add the database we will migrate it later.

**Comment by 01101000011011110110010100001010** [comment]
Also, do you do python scripts. I need something that will convert a CVS file to JSON files.

**Comment by nresh** [comment]
I've made the updates, you can check on the same url https://3-styling-and-asset-updates-to-landing-page.dqlawzyoy6p51.amplifyapp.com/

> Also, add the database we will migrate it later.

ok can you make a new issue/bounty for this?  creating a new db, and connecting a form to populate it is beyond the scope of this issue and PR #4 (i.e. beyond the scope of creating/updating a static site).

> Also, do you do python scripts. I need something that will convert a CVS file to JSON files.

yes I know Python - did you mean CSV (comma separated) rather than CVS?  can you give more details on this, perhaps in another issue and/or repo?  for example, can I get a sample input CSV file and output JSON file.

also, after you're happy with the content/layout and merge the PR - would you mind marking the two bounties as "Work Done" on gitcoin: 
- https://gitcoin.co/issue/Harems-io/Front-end/1/100026098
- https://gitcoin.co/issue/Harems-io/Front-end/3/100026105

**Comment by 01101000011011110110010100001010** [comment]
@nresh I'm still got a gitcoin support ticket open for the first one. Is this one marked as done?

**Comment by nresh** [comment]
@01101000011011110110010100001010 yep I see that it's marked as "Work Done" thanks!

btw, you can close the support ticket for the first one - I received the ETH for that one.  the reason it seemed like it wasn't working was cause the original transaction was dropped and replaced, but I actually got the ETH, so everything worked out fine.

btw, I actually prefer to be paid to my ETH address rather than through paypal.  the reason is that paypal is forcing me to create a seller account and puts a hold on the funds for a while. 

so if you don't mind, for future work, can you try to use ETH like you did for the first issue?

**Comment by nresh** [comment]
also, an alternative for payment is I can send you Quickbooks invoices when I complete work and you can pay me through my linked bank account on Quickbooks.  the process could be: you describe the work, and I give you a price estimate.  and then when I complete the work, I send you the invoice and you pay it thru the Quickbooks portal.

I guess it depends on how much more work you got for me - if this will be an ongoing thing and there is a decent amount more to do, then maybe invoices are better.  but if this is temporary and just a small amount of additional work, then I prefer to go through Gitcoin and use my ETH address.

**Comment by 01101000011011110110010100001010** [comment]
@nresh I want to pay in crypto also let me see if I can get my account fix today. I will post that python task in a little.

**Comment by 01101000011011110110010100001010** [comment]
@nresh I got some work for you let me know if you can do it?

**Comment by nresh** [comment]
@01101000011011110110010100001010 yes i should be able to do it in the next few days - what specifically do you need?

**Comment by 01101000011011110110010100001010** [comment]
@nresh  https://github.com/Harems-io/Rarity-tools-clone/issues/1
